src/apps/vpn/cmake/macos.cmake

```diff
@@ -85,15 +85,7 @@ include(${CMAKE_SOURCE_DIR}/scripts/cmake/osxtools.cmake)
 include(${CMAKE_SOURCE_DIR}/scripts/cmake/golang.cmake)
 include(${CMAKE_SOURCE_DIR}/scripts/cmake/rustlang.cmake)
 
-# Enable Balrog for update support.
```

Comment:
Ignore - that's my totally unrelated struggle to get balrog to work on my machine